.github/workflows/assignment-3_ckappeswordleclone.yml

```diff
@@ -56,7 +56,7 @@ jobs:
         id: deploy-to-webapp
         uses: azure/webapps-deploy@v2
         with:
-          app-name: 'Wordle2023'
+          app-name: 'CKappesWordleClone'
           slot-name: 'Production'
-          publish-profile: ${{ secrets.AZUREAPPSERVICE_PUBLISHPROFILE_API }}
+          publish-profile: ${{ secrets.AZUREAPPSERVICE_PUBLISHPROFILE_2DA3D9F5A4AB4A339FFC12D19747F4D4 }}

```

Comment:
Create a secret in your repo that matches the name in his original file (AZUREAPPSERVICE_PUBLISHPROFILE_API).
Let me know if you have questions about how to do this, I can help. 
Same thing goes for the yml file for your front end. This is so that when you are making a pull request the workflows pass.

Reply:
Will do.